spacemacs.org

```diff
@@ -0,0 +1,97 @@
+* Why I use Spacemacs
+
+Software engineering is a craft, and one of the most important thing to any
+craftman (or craftwoman!), is their tools. Editor choice is a very personal and
+[[https://en.wikipedia.org/wiki/Editor_war][controversial topic]]. I don't want to prescribe that anyone should use what I
+use, but I do want to add my two cents as to why I'm incredibly happy with the
+choice I made.
+
+I use a project called Spacemacs as my main editor for almost any task. I
+really love it. I actually contribute [[https://salt.bountysource.com/teams/spacemacs/supporters][a little every month]] to help fund its
+development.
+
+** What is Spacemacs?
+
+[[http://spacemacs.org/][Spacemacs]] is a free and open-source Emacs configuration. It pulls together the
+best of Vim and Emacs, and it does so with a focus on ease of use and
+productivity.
+
+I had been thinking about trying either Vim or Emacs for a while. I liked
+the idea of Vim's keybinding and learning them so I could have a text editor
+available on whatever machine I use.
+
+But I had also heard a ton about how many great things you can do completely
```

Comment:
Maybe simplify this line's phrasing of too consecutive of a structure.
"But I had also repeatedly heard how many great things you can do completely inside Emacs."
Hopefully didn't deviate from your intended meaning much. Maybe also `inside -> in/from`, if you want.